### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by silversword05**
Added the blocks and changed the messages in the end and play screen.
Please change the address path as required.

### CODE REVIEW

File: Snakes.py [hunk 1]
```diff
@@ -8,15 +8,16 @@
 gameDisplay = pygame.display.set_mode((800,600))
 
 pygame.display.set_caption('SNAKES')
-img=pygame.image.load('Assets\Images\head.png')
```
Comment (jajodiaraghav) [inline Snakes.py:11]: Why are you modifying this line. I don't think this will work

**Comment by jajodiaraghav** [comment]
The work looks great :)
@AadiCool I think a choice should be provided to user whether they want the blocks to appear or they want to play the default one.

This can be done either putting this in separate world, or giving an option in settings page to change this behavior. What do you think?

File: Snakes.py (hunk 1 above)
Comment (silversword05) [reply Snakes.py:11]: It is difficult for me to set up paths every time. So I keep the images in the folder containing the py file.

**Comment by silversword05** [comment]
With due respect, Sir this will make the game quite easy. So I would prefer not to be.

File: Snakes.py (hunk 1 above)
Comment (silversword05) [reply Snakes.py:11]: Sir, It at least works on my machine.

**Comment by jajodiaraghav** [comment]
> It is difficult for me to set up paths every time. So I keep the images in the folder containing the py file.

I see what the problem is. Did you rebase the latest master changes on your local branch?

File: Snakes.py [hunk 2]
```diff
@@ -139,16 +160,28 @@ def gameLoop():
...
+                sound = pygame.mixer.Sound("Point.wav")
+                sound.play()
+
         if len(snakeList)>snakeLen:
             del snakeList[0]
         for body in snakeList[:-1]:
             if body==snakeHead:
                 pyOver=True
+                lossreason ='Oooops You Hit YOURSELF'
+                sound = pygame.mixer.Sound("Point.wav")
```
Comment (jajodiaraghav) [inline Snakes.py:183]: Please rebase and then change this path to the proper file present in assets folder

File: Snakes.py [hunk 3]
```diff
@@ -157,10 +190,13 @@ def gameLoop():
 
         if foodX==x and foodY==y:
             foodX=round(random.randrange(0,790)/10.0)*10.0
-            foodY=round(random.randrange(0,590)/10.0)*10.0
+            foodY=round(random.randrange(20,590)/10.0)*10.0
             snakeLen+=1
-            sound=pygame.mixer.Sound("Assets\Sounds\Point.wav")
+            blocks(foodX,foodY,snakeLen-1)
+            sound=pygame.mixer.Sound("Point.wav")
```
Comment (jajodiaraghav) [inline Snakes.py:196]: Please rebase and then change this path to the proper file present in assets folder

File: Snakes.py [hunk 4]
```diff
@@ -71,6 +72,23 @@ def snake(snakeList):
...
+        if abs(blockX-foodX)<=20 or abs(blockY-foodY)<=20 :
+            i-=1
+            continue
+
+        block.append([blockX,blockY])
+    return block
+
+
+
+
```
Comment (jajodiaraghav) [inline Snakes.py:91]: Please decrease the line breaks here to just 2

File: Snakes.py [hunk 5]
```diff
@@ -8,15 +8,16 @@
...
+img=pygame.image.load('head.png')
 pygame.display.set_icon(img)
 dirn="right"
 clock = pygame.time.Clock()
 font= pygame.font.SysFont("comicsansms", 35)
 k=25
 def score(score):
-	t=font.render("Score: "+str(score), True, (0,0,0))
-	gameDisplay.blit(t,[0,0])
+    s = "            PRESS P TO PAUSE"
```
Comment (jajodiaraghav) [inline Snakes.py:18]: This is not the right way. Remove the space before the message. and then render is properly using the pos passed as argument in the font.render function

File: Snakes.py [hunk 6]
```diff
@@ -83,17 +101,20 @@ def gameLoop():
...
             image = pygame.image.load('python.jpg')        
             gameDisplay.blit(image,(0,0))
             message("Game Over! Press C to play Again, Q to Quit",(255,0,0),30)
+            message(lossreason, (255, 0, 0), 55)
             pygame.display.update()
 
             for event in pygame.event.get():
                 if event.type==pygame.KEYDOWN:
-                    if event.key==pygame.K_q:
+                    if event.key==pygame.K_q :
```
Comment (jajodiaraghav) [inline Snakes.py:117]: Remove the space after `K_q`

File: Snakes.py (hunk 1 above)
Comment (jajodiaraghav) [reply Snakes.py:11]: > Sir, It at least works on my machine.

This simply means that the issue is in your end. You must be doing something wrong.

**Comment by jajodiaraghav** [comment]
I can see a lot of issues with this PR. This is due to lack of experience using git. I would request you to go through the list I sent in the gitter channel. You did not pull and rebase your local with upstream.
Also, The commit message and issue message says "Add files via upload". Can you please explain?

I would be happy to merge this PR. But I cannot do so until you fix the issues. Otherwise it would break the entire app.

**Pull request closed without merging**